### PR TITLE
[DOCS-1700] add new prometheus metrics collection method

### PR DESCRIPTION
**Pull request by cswatt**
### What does this PR do?
Adds another method of collecting metrics (with native Prometheus annotations)

### Motivation
PM request

### Preview

https://docs-staging.datadoghq.com/cswatt/k8s-prom/agent/kubernetes

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by cohenyair — APPROVED** [review]

**Comment by cswatt** [comment]
Hold for release of agent 7.26

**Review by ahmed-mez — APPROVED** [review]
Thank you! Agent 7.26 is released and the helm chart is updated as well. This can be merged 🎉

File: content/en/agent/kubernetes/prometheus.md [hunk 1]
```diff
@@ -131,11 +131,44 @@ For a full list of available parameters for instances, including `namespace` and
 
     {{< img src="integrations/guide/prometheus_kubernetes/prometheus_collected_metric_kubernetes.png" alt="Prometheus metric collected kubernetes">}}
 
+## Metric collection with Prometheus annotations
+
+With Prometheus Autodiscovery, the Datadog Agent is able to detect native Prometheus annotations (for example: `prometheus.io/scrape`, `prometheus.io/path`, `prometheus.io/port`) and schedule OpenMetrics checks automatically to collect Prometheus metrics in Kubernetes.
+
+### Requirements
+
+- Datadog Agent 7 or 6.26+ (for Pod checks)
```
Comment (ruthnaebeck) [inline content/en/agent/kubernetes/prometheus.md:140]:
```suggestion
- Datadog Agent v7 or v6.26+ (for Pod checks)
```

File: content/en/agent/kubernetes/prometheus.md [hunk 2]
```diff
@@ -131,11 +131,44 @@ For a full list of available parameters for instances, including `namespace` and
...
+- Datadog Cluster Agent 1.11+ (for service and endpoint checks)
```
Comment (ruthnaebeck) [inline content/en/agent/kubernetes/prometheus.md:141]:
```suggestion
- Datadog Cluster Agent v1.11+ (for service and endpoint checks)
```

**Review by ruthnaebeck — CHANGES_REQUESTED** [review]

**Review by ruthnaebeck — APPROVED** [review]

**Pull request merged**